Formula/chromaprint.rb

```diff
@@ -16,8 +16,16 @@ class Chromaprint < Formula
   depends_on "ffmpeg"
 
   def install
-    system "cmake", "-DCMAKE_BUILD_TYPE=Release", "-DBUILD_TOOLS=ON", ".", *std_cmake_args
-    system "make", "install"
+    args = %W[
+      -DCMAKE_BUILD_TYPE=Release
```

Comment:
We don't need this anymore; it should be part of `std_cmake_args`.

Reply:
Ok I’ll fix that and repush

Reply:
Fixed